Formula/k3d.rb

```diff
@@ -22,31 +22,35 @@ class K3d < Formula
   depends_on "go" => :build
 
   def install
+    require "net/http"
+    uri = URI("https://update.k3s.io/v1-release/channels")
+    resp = Net::HTTP.get(uri)
+    resp_json = JSON.parse(resp)
+    k3s_version = resp_json["data"].find { |channel| channel["id"]=="stable" }["latest"].sub("+", "-")
+
     system "go", "build",
            "-mod", "vendor",
-           "-ldflags", "-s -w -X github.com/rancher/k3d/v#{version.major}/version.Version=v#{version}"\
-                       " -X github.com/rancher/k3d/v#{version.major}/version.K3sVersion=latest",
-           "-trimpath", "-o", bin/"k3d"
+           *std_go_args(ldflags: "-s -w -X github.com/rancher/k3d/v#{version.major}/version.Version=v#{version} -X github.com/rancher/k3d/v#{version.major}/version.K3sVersion=#{k3s_version}")
```

Comment:
Can you define the ldflags in a variable above and just reference that instead? This is harder to read

Reply:
:heavy_check_mark: